prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseDetailConfiguration.cs

```diff
@@ -6,6 +7,9 @@ namespace Prime.Configuration.Database
 {
     public class LicenseDetailConfiguration : SeededTable<LicenseDetail>
     {
+        public static readonly DateTimeOffset IMREG_UPDATE1_DATE = DateTimeOffset.Parse("2022-02-01 -7:00");
```

Comment:
The convention we have been moving toward is to not use screaming snake case and instead use the regular `PascalCase`, even for constants. I acknowledge that `SEEDING_DATE` is still in the old style, but please use the newer convention for new code.